apps/web/playwright/payment-apps.e2e.ts

```diff
@@ -55,9 +53,7 @@ test.describe("Payment app", () => {
     const user = await users.create();
     await user.apiLogin();
     const paymentEvent = user.eventTypes.find((item) => item.slug === "paid");
-    if (!paymentEvent) {
```

Comment:
Since we're not type narrowing can we at least assert paymentEvent! as not null? To avoid using paymentEvent?.  everywhere